vaadin-map-flow-parent/vaadin-map-flow/src/main/resources/META-INF/resources/frontend/vaadin-map/mapConnector.js

```diff
@@ -35,6 +35,26 @@ openLayersSetUserProjection('EPSG:4326');
 
           synchronize(change, context);
         });
+      },
+      /**
+       * Forces a render of the OpenLayers map. Some objects in OpenLayers are not observable
+       * and do not trigger change events, for example Style objects or any of their children.
+       * In these cases this method can be called from the synchronization functions of these
+       * objects.
+       * This method will trigger a debounced render of the map by firing a change event from
+       * each layer. We simply render all layers as a sync. function does not know which layer
+       * its synced object is in. Even if the change event is fired from multiple layers, this
+       * only results in a single render of the map.
+       */
+      forceRender() {
+        if (this._forceRenderTimeout) return;
```

Comment:
nit: we don't seem to have any lint rules for the JS part of the project. On the web-component project, we have the [curly rule](https://eslint.org/docs/latest/rules/curly) that would fail here with `Expected { after 'if' condition`.

Reply:
Updated the early return to use curly braces.